### ISSUE TITLE
Nav - accessibility - logo has invalid aria markup of role=presentation 

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#17]
instead of  role=presentation it should have aria-hidden=true

see screenshot for code in question 

![Nav_-_accessibility_-_logo_has_invalid_aria_markup_of_role=presentation](https://user-images.githubusercontent.com/2722522/64774464-28c8f480-d54c-11e9-8e9a-e562538c3965.png)

**Comment by Dan-KeduTesting** [comment]
passed testing
